android/gradle.properties

```diff
@@ -1,7 +1,6 @@
 propMinSdkVersion=21
 propTargetSdkVersion=34
 propCompileSdkVersion=34
-propBuildToolsVersion=34.0.0
```

Comment:
What are the benefits of this change? Manually specifying the latest tools version is more flexible.

Reply:
The Gradle plugin already has the latest version hard-coded. For example, it installs 34.0.2 instead of the current 34.0.0. Androdi Studio complains that `buildToolsVersion` is deprecated.

https://stackoverflow.com/questions/69279770/buildtoolsversion-and-compilesdkversion-had-been-deprecated